Runtime-extra-platforms failing in DNNE compilation

```
Building native export: "C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\bin\HostArm64\arm64\cl.exe" /O2 /LD /TC /MT /GS /Zi /D DNNE_ASSEMBLY_NAME=Microsoft.Interop.Tests.NativeExports /D DNNE_COMPILE_AS_SOURCE /I "C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\include" /I "D:\a\_work\1\s\.packages\dnne\2.1.1\tools\platform" /I "D:\a\_work\1\s\.packages\microsoft.netcore.app.host.win-arm64\11.0.0-preview.3.26170.106\runtimes\win-arm64\native" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\shared" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\um" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\ucrt"  "D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\Microsoft.Interop.Tests.NativeExports.g.c" "D:\a\_work\1\s\.packages\dnne\2.1.1\tools\platform\platform.c" /link /DLL /LIBPATH:"C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\lib\arm64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.26100.0\um\arm64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.26100.0\ucrt\arm64" "D:\a\_work\1\s\.packages\microsoft.netcore.app.host.win-arm64\11.0.0-preview.3.26170.106\runtimes\win-arm64\native\libnethost.lib" Advapi32.lib /IGNORE:4099 /IMPLIB:"D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\bin\Microsoft.Interop.Tests.NativeExportsNE.lib" /OUT:"D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\bin\Microsoft.Interop.Tests.NativeExportsNE.dll" 
  '"C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\bin\HostArm64\arm64\cl.exe"' is not recognized as an internal or external command,
  operable program or batch file.
##[error].packages\dnne\2.1.1\build\DNNE.targets(257,5): error MSB3073: (NETCORE_ENGINEERING_TELEMETRY=Build) The command ""C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\bin\HostArm64\arm64\cl.exe" /O2 /LD /TC /MT /GS /Zi /D DNNE_ASSEMBLY_NAME=Microsoft.Interop.Tests.NativeExports /D DNNE_COMPILE_AS_SOURCE /I "C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\include" /I "D:\a\_work\1\s\.packages\dnne\2.1.1\tools\platform" /I "D:\a\_work\1\s\.packages\microsoft.netcore.app.host.win-arm64\11.0.0-preview.3.26170.106\runtimes\win-arm64\native" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\shared" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\um" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\ucrt"  "D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\Microsoft.Interop.Tests.NativeExports.g.c" "D:\a\_work\1\s\.packages\dnne\2.1.1\tools\platform\platform.c" /link /DLL /LIBPATH:"C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\lib\arm64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.26100.0\um\arm64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.26100.0\ucrt\arm64" "D:\a\_work\1\s\.packages\microsoft.netcore.app.host.win-arm64\11.0.0-preview.3.26170.106\runtimes\win-arm64\native\libnethost.lib" Advapi32.lib /IGNORE:4099 /IMPLIB:"D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\bin\Microsoft.Interop.Tests.NativeExportsNE.lib" /OUT:"D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\bin\Microsoft.Interop.Tests.NativeExportsNE.dll" " exited with code 9009.
D:\a\_work\1\s\.packages\dnne\2.1.1\build\DNNE.targets(257,5): error MSB3073: The command ""C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\bin\HostArm64\arm64\cl.exe" /O2 /LD /TC /MT /GS /Zi /D DNNE_ASSEMBLY_NAME=Microsoft.Interop.Tests.NativeExports /D DNNE_COMPILE_AS_SOURCE /I "C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\include" /I "D:\a\_work\1\s\.packages\dnne\2.1.1\tools\platform" /I "D:\a\_work\1\s\.packages\microsoft.netcore.app.host.win-arm64\11.0.0-preview.3.26170.106\runtimes\win-arm64\native" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\shared" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\um" /I "C:\Program Files (x86)\Windows Kits\10\Include\10.0.26100.0\ucrt"  "D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\Microsoft.Interop.Tests.NativeExports.g.c" "D:\a\_work\1\s\.packages\dnne\2.1.1\tools\platform\platform.c" /link /DLL /LIBPATH:"C:\Program Files\Microsoft Visual Studio\18\Enterprise\VC\Tools\MSVC\14.51.36014\lib\arm64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.26100.0\um\arm64" /LIBPATH:"C:\Program Files (x86)\Windows Kits\10\Lib\10.0.26100.0\ucrt\arm64" "D:\a\_work\1\s\.packages\microsoft.netcore.app.host.win-arm64\11.0.0-preview.3.26170.106\runtimes\win-arm64\native\libnethost.lib" Advapi32.lib /IGNORE:4099 /IMPLIB:"D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\bin\Microsoft.Interop.Tests.NativeExportsNE.lib" /OUT:"D:\a\_work\1\s\artifacts\obj\NativeExports\Release\net11.0\win-arm64\dnne\bin\Microsoft.Interop.Tests.NativeExportsNE.dll" " exited with code 9009. [D:\a\_work\1\s\src\libraries\System.Runtime.InteropServices\tests\TestAssets\NativeExports\NativeExports.csproj]
  baseline ready
  got a change

```

## Build Information
Build: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/results?buildId=1395131&view=logs&j=542c931e-8293-51c7-dbad-a0a58a58ddc4
Build error leg or test failing: Build / windows-arm64 Release Libraries_Release_CoreCLR / Build product
Pull request: https://github.com/dotnet/runtime/pull/127387
<!-- Error message template  -->
## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md).

<!-- Use ErrorMessage for String.Contains matches. Use ErrorPattern for regex matches (single line/no backtracking). Set BuildRetry to `true` to retry builds with this error. Set ExcludeConsoleLog to `true` to skip helix logs analysis. -->

```json
{
  "ErrorMessage": "",
  "ErrorPattern": "HostArm64.*cl.exe.*is not recognized as an",
  "BuildRetry": false,
  "ExcludeConsoleLog": false
}
```


<!-- Known issue validation start -->
 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=1395131
**Error message validated:** `[HostArm64 cl.exe is not recognized as an`]
**Result validation:** :x: Known issue did not match with the provided build.
**Validation performed at:** 4/24/2026 11:14:25 PM UTC
<!-- Known issue validation end -->
<!--Known issue error report start -->

### Report
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|0|0|
<!--Known issue error report end -->
<!-- Known issue validation start -->
 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=1395131
**Error message validated:** `[HostArm64.*cl.exe.*is not recognized as an`]
**Result validation:** :white_check_mark: Known issue matched with the provided build.
**Validation performed at:** 4/26/2026 11:34:11 PM UTC
<!-- Known issue validation end -->
<!--Known issue error report start -->

### Report
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|0|0|
<!--Known issue error report end -->
<!--Known issue error report start -->

### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[1404470](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1404470)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1404470/logs/433)|dotnet/runtime#127613|
|[1404374](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1404374)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1404374/logs/212)||
|[1403445](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1403445)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1403445/logs/1031)|dotnet/runtime#127498|
|[1403305](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1403305)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1403305/logs/212)||
|[1403076](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1403076)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1403076/logs/361)||
|[1402716](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1402716)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1402716/logs/199)||
|[1402544](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1402544)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1402544/logs/278)|dotnet/runtime#127578|
|[1401195](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1401195)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1401195/logs/1220)||
|[1401527](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1401527)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1401527/logs/198)||
|[1400782](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1400782)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1400782/logs/166)||
|[1400780](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1400780)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1400780/logs/349)||
|[1400724](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1400724)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1400724/logs/210)||
|[2962350](https://dev.azure.com/dnceng/internal/_build/results?buildId=2962350)|dotnet-runtime|Build product|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2962350/logs/620)||
|[1399604](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1399604)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1399604/logs/489)|dotnet/runtime#127498|
|[2962114](https://dev.azure.com/dnceng/internal/_build/results?buildId=2962114)|dotnet-runtime|Build product|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2962114/logs/603)||
|[1399510](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1399510)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1399510/logs/209)||
|[1399212](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1399212)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1399212/logs/667)||
|[1398754](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1398754)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1398754/logs/210)||
|[1398689](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1398689)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1398689/logs/376)|dotnet/runtime#127465|
|[1397706](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1397706)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1397706/logs/200)||
|[1397424](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1397424)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1397424/logs/610)||
|[2960695](https://dev.azure.com/dnceng/internal/_build/results?buildId=2960695)|dotnet-runtime|Build product|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2960695/logs/2981)||
|[1397304](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1397304)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1397304/logs/227)||
|[1395131](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1395131)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1395131/logs/382)|dotnet/runtime#127387|
|[1394339](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1394339)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/1394339/logs/634)||
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|4|24|25|
<!--Known issue error report end -->

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -134,7 +134,7 @@
     <NETStandardLibraryRefVersion>2.1.0</NETStandardLibraryRefVersion>
     <NetStandardLibraryVersion>2.0.3</NetStandardLibraryVersion>
     <MicrosoftDiagnosticsToolsRuntimeClientVersion>1.0.4-preview6.19326.1</MicrosoftDiagnosticsToolsRuntimeClientVersion>
-    <DNNEVersion>2.1.1</DNNEVersion>
+    <DNNEVersion>2.1.2</DNNEVersion>
     <MicrosoftBuildVersion>17.11.48</MicrosoftBuildVersion>
     <MicrosoftBuildTasksCoreVersion>17.11.48</MicrosoftBuildTasksCoreVersion>
     <MicrosoftBuildFrameworkVersion>17.11.48</MicrosoftBuildFrameworkVersion>

